nntrainer/src/conv2d_layer.cpp

```diff
@@ -187,8 +159,6 @@ sharedConstTensor Conv2DLayer::backwarding(sharedConstTensor derivative,
              input_dim.width() + padding[1] * 2);
   ret.setZero();
 
-#ifdef USE_BLAS
-  // Calculate DelK, DelBias
```

Comment:
Maybe unintended deletion?

Reply:
No, No need the previous implementation.